aleph/migrate/versions/c52a1f469ac7_create_bookmark_table.py

```diff
@@ -0,0 +1,33 @@
+"""create bookmark table
+
+Revision ID: c52a1f469ac7
+Revises: 274270e01613
+Create Date: 2023-01-30 08:53:59.370110
+
+"""
+
+# revision identifiers, used by Alembic.
+from alembic import op
+import sqlalchemy as sa
+
+revision = "c52a1f469ac7"
+down_revision = "274270e01613"
+
+
+def upgrade():
+    op.create_table(
+        "bookmark",
+        sa.Column("id", sa.Integer(), primary_key=True),
+        sa.Column("created_at", sa.DateTime(), nullable=False),
+        sa.Column("role_id", sa.Integer(), sa.ForeignKey("role.id"), nullable=False),
+        sa.Column("entity_id", sa.String(length=128), nullable=False),
```

Comment:
Maybe we should also track the `collection_id` the entity belongs to here? This would allow us to filter by collection efficiently (a use case we will likely want to support in the future, but not 100% sure) and it would allow us to select only bookmarks in collections a user has access to (see https://github.com/alephdata/aleph/pull/2843/files#r1092250155).

This may be a bit of a premature optimization, but would save us the manualy work to populate this information for existing bookmarks in the future.